python/lsst/analysis/ap/zooniverseCutouts.py

```diff
@@ -46,9 +46,11 @@ class ZooniverseCutoutsConfig(pexConfig.Config):
         default=30,
     )
     urlRoot = pexConfig.Field(
-        doc="URL that the resulting images will be served to Zooniverse from, for the manifest file.",
+        doc="URL that the resulting images will be served to Zooniverse from, for the manifest file. "
+            "If not set, no manifest file will be written.",
```

Comment:
It looks like there should be a \ here at the end of line 49 to indicate that the string for doc continues onto line 50.
```suggestion
        doc="URL that the resulting images will be served to Zooniverse from, for the manifest file. " \
            "If not set, no manifest file will be written.",
```

Reply:
Because this is in an assignment statement, the string is automatically combined, no line-continuation or parentheses necessary.